### PR TITLE
report stop event while `close/disconnect/idle/stop` case

### PR DESCRIPTION
**Pull request by PixMod**
fix bug: https://bug.rokid-inc.com/zentaopms/www/index.php?m=bug&f=view&bugID=22777

 * [x] npm lint check

### CODE REVIEW

File: bluetooth-music/app.js [hunk 1]
```diff
@@ -138,6 +139,14 @@ function handleUrl (url) {
       }
       uploadEvent(protocol.AUDIO_STATE.PLAYING)
       break
+    case '/STOPPED':
```
Comment (yorkie) [inline bluetooth-music/app.js]: This is inconsistent with others.

**Review by yorkie — CHANGES_REQUESTED** [review]
We should keep the Yoda URL names being consistent, and documentation is also required if something is new :)

File: bluetooth-music/app.js (hunk 1 above)
Comment (PixMod) [reply bluetooth-music/app.js]: This is private URL, how to separate it from public URL?

Comment (legendecas) [reply bluetooth-music/app.js]: Conventionally, we would name the url with prefix `_` and keep the rest of the name consistent with other urls. Or made the prefix explicit, e.g. `/private/stopped` or `/event/stopped`.

Comment (PixMod) [reply bluetooth-music/app.js]: > Conventionally, we would name the url with prefix `_` and keep the rest of the name consistent with other urls. Or made the prefix explicit, e.g. `/private/stopped` or `/event/stopped`.

Good instruction.

Comment (yorkie) [reply bluetooth-music/app.js]: Why do we have the private URLs?

Comment (PixMod) [reply bluetooth-music/app.js]: Due to 

> Why do we have the private URLs?

2 reasons:
 - When bluetooth-music app is not launched, user can play music in phone, so the /PLAYING event can only be received by bluetooth-service, so we need forward this event to music-app as private URL.
 - In this bug, when directly close bluetooth when playing, there's only `close` event while no `stop` event, so we need send a mock `STOPPED` event to music-app by private URL.

**Review by yorkie — APPROVED** [review]

**Review by legendecas — APPROVED** [review]

**Pull request merged**